javaslang/generator/Generator.scala

```diff
@@ -122,6 +126,35 @@ def generateMainClasses(): Unit = {
         """
       }
 
+      def genMapAliases(mapType: String, name: String): String = {
+        xs"""
+          /$javadoc
+           * Alias for {@link $mapType#of(Object, Object)}
```

Comment:
Yes, I remember correctly: IDEA accept JavaDoc link as `{@link HashMap#of(K, V)}`, but javadoc linter in stage `javadoc:javadoc` generate warnings `warning - Tag @link: can't find of(K, V) in javaslang.collection.HashMap` for all links to methods with generic classes in arguments.